packages/cli/src/commands/proxy/connect.ts

```diff
@@ -16,6 +16,10 @@ export default class Connect extends ProfileCommand<typeof Connect> {
     ...tunnelServerFlags,
     ...urlFlags,
     ...ux.table.flags(),
+    'env-id': Flags.string({
```

Comment:
Changed to id.
Added `env-id` as flag alias, it might be worth adding the alias to other commands and in the future switch between the alias and the flag as ID can be ambiguous